README.md

```diff
@@ -1,3 +1,6 @@
+## Cognoml is inactive since deciding to return Jupyter Notebooks in April 2017
```

Comment:
Can you just put these two sentences in bold underneath the first header? I don't think they really belong as headers.